Mathlib/Data/Set/Operations.lean

```diff
@@ -143,7 +147,7 @@ This function is more flexible than `f '' univ`, as the image requires that the
 and not an arbitrary Sort. -/
 def range (f : ι → α) : Set α := {x | ∃ y, f y = x}
 
-@[simp] theorem mem_range {x : α} : x ∈ range f ↔ ∃ y, f y = x := Iff.rfl
+@[simp, grind =] theorem mem_range {x : α} : x ∈ range f ↔ ∃ y, f y = x := Iff.rfl
 
```

Comment:
Is is actually useful to tag lemmas like this? My understanding was that grind couldn't really deal with the existential quantifier on the RHS anyway.

Reply:
It won't *prove* the existential quantifier, but it will *use* it.